ui/models/job.js

```diff
@@ -140,17 +140,34 @@ export default class JobModel {
 
         TaskclusterModel.load(decisionTaskId).then(async results => {
           const actionTaskId = slugid();
-          const addNewJobsTask = results.actions.find(
-            action => action.name === 'add-new-jobs',
+          const taskLabels = value.map(job => job.job_type_name);
+
+          let retriggerAction = results.actions.find(
+            action => action.name === 'retrigger-multiple',
           );
+          let actionInput = {
+            requests: [{ tasks: taskLabels }],
+          };
+          if (!retriggerAction) {
+            // The `retrigger-multiple` action as introduced in Bug 1521032, to all the action
+            // to control whether new task are created, or existing ones re-run. We fall back
+            // to `add-new-jobs` to support pushing old revision to try, where the duplicating
+            // the release tasks impacted is unlikely to cause problems.
```

Comment:
Confirming I'm understanding this: Are these backwards compat changes so we can merge this without waiting for the upstream TC patch you mention and to support actions on older revisions that wouldn't be affected by said upstream bug?

Reply:
Just the later. I'd like to get support for the new way landed on at least mozilla-central before landing this, in case the input format changes.

The bug will still affect old revision, but the impact on them is going to be minor.

Reply:
Thanks for clarifying. Let us know when this is ready to be merged.